projects/canopy/src/lib/card/card-navigation-title/card-navigation-title.component.html

```diff
@@ -0,0 +1,6 @@
+<lg-heading [level]="headingLevel" *ngIf="link && title && headingLevel">
```

Comment:
I think instead of adding the `*ngIf="link && title && headingLevel"` we could add a `console.error` within the `onInit` so that users are aware of what went wrong instead of silently have it not appearing?

Reply:
I think we still need to be defensive in our coding to allow the component to fail gracefully.... i take your point in terms of logging errors. It may be useful but perhaps one for a wider discussion as I'm sure there are other components in a similar boat that this type of approach could be considered?

Reply:
i've added some logging but as I mentioned before I think handling the missing inputs gracefully is better than letting it just break